core/src/main/scala/com/softwaremill/react/kafka/ReactiveKafka.scala

```diff
@@ -95,15 +96,67 @@ class ReactiveKafka {
     Sink.fromGraph(new KafkaGraphStageSink(producer))
   }
 
-  def graphStageSource[K, V](props: ConsumerProperties[K, V]) = {
+  // Scala DSL
+  def graphStageSource[K, V](
+    props: ConsumerProperties[K, V]
+  ) = {
     val consumer = ReactiveKafkaConsumer(props)
     Source.fromGraph(new KafkaGraphStageSource(consumer))
   }
 
+  // Scala DSL
+  def graphStageSource[K, V](
+    props: ConsumerProperties[K, V],
+    topicsAndPartitions: List[TopicPartition]
+  ) = {
+    val consumer = ReactiveKafkaConsumer(props, topicsAndPartitions)
+    Source.fromGraph(new KafkaGraphStageSource(consumer))
+  }
+
+  // Scala DSL
+  def graphStageSource[K, V](
+    props: ConsumerProperties[K, V],
+    topicsAndPartitions: List[TopicPartition],
+    topicPartitionOffsetsMap: Map[TopicPartition, Long]
```

Comment:
Just to be clear with my issue with this:
What happens if I specify topicsAndPartitions = List(TopicPartition("t1",0)) and topicPartitionOffsetsMap = Map(TopicPartition("t2",0), 123). Will it consume from both topics? But maybe this was a typo from my part and I want to only consume from t1...

Other than this. thx for this nice contribution